apps/api_web/lib/api_web/views/facility_view.ex

```diff
@@ -73,4 +73,10 @@ defmodule ApiWeb.FacilityView do
       }
     end
   end
+
+  def attribute_set(%{assigns: %{api_version: ver}} = conn) when ver < "2019-07-01" do
+    conn |> super() |> MapSet.put("name")
```

Comment:
- Can we make the new version the first case, and the older versions the fallback?
- I was thinking you should drop `long_name`, but I think that'll break the website?

Reply:
Fixed. Yes, website switched to using `long_name` and they're still using the same (older) API version that they used before, so removing `long_name` would break it for them indeed.